test/simplifyExpression/fractionsSearch/cancelLikeTerms.test.js

```diff
@@ -21,13 +21,13 @@ describe('cancel like terms', function () {
     ['(2x+5)^8 / (2x+5)^2', '(2x + 5)^(8 - (2))'],
     ['(4x^3) / (5x^2)', '(4x^(3 - (2))) / (5)'],
     ['-x / -x', '1'],
+    ['(6x)/ x', '(6)'],
+    ['(2 * x^2 * 2x) / (y * y)', '(2x^2 * 2x) / (y * y)']
   ];
 
   tests.forEach(t => {
     const before = t[0];
     const after = t[1];
-    it(before + ' -> ' + after, function () {
```

Comment:
why'd you get rid of the `it`? the text it prints is helpful

maybe you noticed the problem in #146 ? maybe you could take that off of kevin's hands so he can focus on the parser (in a separate PR though, keep the `it` the way it is here)